src/Microsoft.Identity.Client/Internal/Instance/AdfsAuthority.cs

```diff
@@ -131,7 +131,9 @@ protected override void AddToValidatedAuthorities(string userPrincipalName)
             }
             catch (Exception exc)
             {
-                requestContext.Logger.Info("On-Premise ADFS enrollment server endpoint lookup failed. Error - " + exc.Message);
+                string msg = string.Format("On-Premise ADFS enrollment server endpoint lookup failed. Error - " + exc.Message);
```

Comment:
exc.Message could contain PII. you should run the exception through scrubber and create separate log messages.

Reply:
done.